teachertool/src/components/CriteriaResultEntry.tsx

```diff
@@ -91,9 +112,10 @@ const CriteriaResultToolbarTray: React.FC<{ criteriaId: string }> = ({ criteriaI
     }
 
     async function handleDeleteClickedAsync() {
-        if (confirm(Strings.ConfirmDeleteCriteriaInstance)) {
-            removeCriteriaFromChecklist(criteriaId);
-        }
+        removeCriteriaFromChecklist(criteriaId);
+        const toast = makeToast("info", Strings.CriteriaDeleted);
+        toast.jsx = <UndoDeleteCriteriaButton criteriaId={criteriaId} toastId={toast.id} />;
```

Comment:
In `makeToast`, it looks like you added a parameter for jsx, but it seems like we're not using it here. Was there a specific reason for that?

Reply:
Yes. I originally just had the `<UndoDeleteCriteriaButton/>` inside the make toast. However, in order to have clicking the undo button dismiss the toast, I needed to have access to the toast id so I switched to setting the toast's jsx to the button after making the toast. That said, I could see future scenarios where passing in jsx when making the toast would be helpful so I left it in. I can remove it, though, since it's not getting used with this change.

Reply:
I see. I would probably lean towards removing it in that case, at least for now. Either way is fine though.